web/src/app/pages/tasks/intern/intertasks.controller.js

```diff
@@ -37,7 +37,7 @@
             var userID = {
                 "id": $email
             };
-            $http.post('https://ezh9ingj6l.execute-api.us-east-1.amazonaws.com/dev/getUserTask', userID).then(function(response) {
+            $http.post('https://rsrxpyrrz4.execute-api.us-east-1.amazonaws.com/dev/tasks/getUserTask', userID).then(function(response) {
```

Comment:
API had another URL right? 
Ex: https://github.com/99xt/interns-portal/blob/0c26e458c8d97e4a623c08b4d0afe6b8e765a96f/web/src/app/pages/home/home.controller.js#L38

Move the host URL to a configuration file. Easy for modifications.